quipucords/api/common/util.py

```diff
@@ -105,6 +74,19 @@ def convert_to_boolean(value):
     return False
 
 
+def convert_to_bool_or_none(value):
+    """Convert a string "True" or "False" to boolean.
+
+    If value already is a boolean, returns it untouched.
+    Otherwise, return None.
+    """
+    if isinstance(value, bool):
+        return value
+    if is_boolean(value):
+        return value.lower() == "true"
+    return None
```

Comment:
This reproduces the original logic that was implicit [here](https://github.com/quipucords/quipucords/pull/2740/files#r1767553142)